hedyorg/hedy
# 💻 Implemented "is" and sleep command for Micro:bit for level 2

Added the assignment to a variable from level 2 in Hedy to Micro:bit
Added sleep functionality for Micro:bit

**How to test**

Follow these steps to verify this PR works as intended:
Open Hedy and choose level 2. Assign a variable and then print it. This will generate a hex file that can be uploaded here:
https://python.microbit.org/v/3/reference. 
You should see the correct python code after you upload the file. 


If you're unsure about any of these, don't hesitate to ask. We're here to help!
